mitodl/open-discussions
# Save spam check results

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #3072 

#### What's this PR do?
Saves the result of the latest spam check for each post/comment.

#### How should this be manually tested?
Create a new post and comment.  Check django admin to make sure a `SpamCheckResult` object was created for it with expected attribute values, including `checks= 1`.
Edit the post and comment.  Check again and make sure `checks= 2`.

#### Screenshots (if appropriate)
<img width="1255" alt="Screen Shot 2020-07-23 at 5 06 48 PM" src="https://user-images.githubusercontent.com/187676/88339091-6fa87e80-cd07-11ea-84bb-6d85a44bdb49.png">

<img width="1225" alt="Screen Shot 2020-07-23 at 5 08 58 PM" src="https://user-images.githubusercontent.com/187676/88339108-76cf8c80-cd07-11ea-8838-7b99e44f5fd9.png">
